api/v1alpha1/componentversion_types.go

```diff
@@ -77,18 +77,20 @@ type Signature struct {
 	// signatures.
 	Name string `json:"name"`
 
-	// PublicKey provides a reference to a Kubernetes Secret that contains a public key
+	// PublicKey provides a reference to a Kubernetes Secret of contain a blob of a public key that
 	// which will be used to validate the named signature.
 	PublicKey SecretRef `json:"publicKey"`
-
-	// PublicKeyBlob defines an inlined public key.
-	//+optional
-	PublicKeyBlob []byte `json:"publicKeyBlob,omitempty"`
 }
 
-// SecretRef specifies a reference to a Secret.
+// SecretRef specifies access to a public key for verification.
 type SecretRef struct {
```

Comment:
Yaaah I know. :/ But I wanted to avoid renaming the JSON as to not break anything existing. Do you think it's fine to break it since we don't really have usage outside us right now? :)

Reply:
Let's call it `Ref`?

Reply:
```go

// SecretRef specifies access to a public key for verification.
type SecretRef struct {
	// Ref is a reference to a Secret that contains a public key.
	// +optional
	Ref v1.LocalObjectReference `json:"ref,omitempty"`

	// Values defines an inlined public key.
	// +optional
	Values []byte `json:"value,omitempty"`
}
```

Looks good to me. We'll have to update every ref everywhere. :D